indico/modules/categories/controllers/display.py

```diff
@@ -537,9 +537,13 @@ def _get_calendar_days(self):
     def _get_days(start_dt, end_dt):
         # Return all days in the open-ended interval
         current_dt = start_dt
+        event_tz = current_dt.tzinfo
```

Comment:
we do, it's just easier to fetch it from there, I don't need to pass an extra arg